src/lib/containers/entity_finder/EntityFinderUtils.ts

```diff
@@ -1,10 +0,0 @@
-import { EntityBundleRequest } from '../../utils/synapseTypes'
-
-// Use the same bundle request object across the entity finder so we can take advantage of the shared cache
-export const BUNDLE_REQUEST_OBJECT: EntityBundleRequest = {
-  includeEntity: true,
-  includeEntityPath: true,
-  includeAnnotations: true,
-  includeBenefactorACL: true,
-  includePermissions: true,
-}
```

Comment:
Part of the reason we used a shared request object was to ensure we didn't show certain Entity Badge icons that we wanted to exclude from the finder.

Since those icons can now be hidden via props, and the Entity Finder is using a shared cache, there's no reason to use this. We will just request the full bundle.